tasks/dataset-split/src/customDatasetSplit.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from coretex import currentTaskRun, NetworkDataset, CustomDataset, CustomSample
```

Comment:
lets use currentTaskRun() function only initially to fetch taskRun, and after that if you need to use taskRun object in other functions pass it through arguments of function instead of calling function currentTaskRun() again, or even better if you need only specific properties from taskRun like certain parameter pass only that parameter to that function instead of passing whole object.